cmd/server/main.go

```diff
@@ -54,38 +54,39 @@ var (
 )
 
 func main() {
-	klog.InitFlags(nil)
```

Comment:
The alternative would be to keep the `-v` flag by manually adding it back, and then use it via `mlog.ValidateAndSetKlogLevelAndFormatGlobally` (we would then not add the `--log-level` flag).

Reply:
That's a good idea. I added the flag.

Reply:
> The alternative would be to keep the -v flag by manually adding it back, and then use it via mlog.ValidateAndSetKlogLevelAndFormatGlobally (we would then not add the --log-level flag).

This would work in a backward compatible way.

> That's a good idea. I added the flag.

I see the `-v` flag but the values are `all`, `info`, `debug` and `trace` which is still a backward incompatible change?